src/pybind/CMakeLists.txt

```diff
@@ -60,10 +60,11 @@ add_custom_command(
   COMMENT "-- COPYING NMODL PYTHON FILES --")
 
 # =============================================================================
-# Copy python binding components into build directory
+# Copy python binding components and examples into build directory
 # =============================================================================
 file(GLOB NMODL_PYTHON_HELPER_FILES "${NMODL_PROJECT_SOURCE_DIR}/nmodl/*.py")
 file(COPY ${NMODL_PYTHON_HELPER_FILES} DESTINATION ${CMAKE_BINARY_DIR}/lib/python/nmodl/)
+file(COPY ${NMODL_PROJECT_SOURCE_DIR}/nmodl/ext DESTINATION ${CMAKE_BINARY_DIR}/lib/python/nmodl/)
```

Comment:
do i misunderstand or you copy _everything_ from `ext` ? seems overkill.

Reply:
note that this `ext` directory is provided by us containing sample mod files and html/css file to run [examples documented on README](https://github.com/BlueBrain/nmodl/blob/master/README.md#using-the-python-api). So we don't copy anything extra from build here.

Reply:
ah `nmodl/ext` vs. `ext` - missed that (and naming should be fixed). This is done in another branch, but we'll merge that when we get to that PR.